src/types/gecko-profile.js

```diff
@@ -115,6 +125,12 @@ export type GeckoSampleStructWithResponsiveness = {|
   stack: Array<null | IndexIntoGeckoStackTable>,
   time: Milliseconds[],
   responsiveness: Array<?Milliseconds>,
+  // CPU usage value of the current thread. Its values are null only if the back-end
+  // fails to get the CPU usage from operating system.
+  // It's landed in Firefox 86, and is only optional because older profile
+  // versions may not have it. No upgrader was written for this change because
```

Comment:
What happens with the CPU feature is not enabled? My understanding is that the property won't be here either